applications/GeoMechanicsApplication/tests/test_dynamic/test_constant_point_load_2d_axi/README.md

```diff
@@ -0,0 +1,22 @@
+# Test Cases for Dynamic solution
+
+**Author:** [Mohamed Nabi](https://github.com/mnabideltares)
+
+**Source files:** [Dynamic solution](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_dynamic/test_constant_point_load_2d_axi)
+
+## Case Specification
```

Comment:
Please add a reference to Verruijt in this documentation.

Further, I think that a remark about the infinite half space of Verruijt is different from the finite domain in this computation. Therefore the solution can not be compared anymore when waves reflect on the boundaries of the model.